vmngclient/dataclasses.py

```diff
@@ -316,6 +316,8 @@ class TenantInfo(DataclassBase):
     organization_name: str = field(metadata={FIELD_NAME: "orgName"})
     sub_domain: str = field(metadata={FIELD_NAME: "subDomain"})
     id: str = field(metadata={FIELD_NAME: "tenantId"})
+    description: Optional[str] = field(default="", metadata={FIELD_NAME: "desc"})
+    state: Optional[str] = field(default="", metadata={FIELD_NAME: "state"})
```

Comment:
Don't we use `None` as default? Or in this case empty string is needed?

`FIELD_NAME` is not needed when the value is the same as name (in regard of `state`)

Reply:
Both are True, it should be `None` and `FIELD_NAME` is obsolete for `state`